src/main/scala/io/kf/etl/processors/download/transform/utils/EntityDataRetriever.scala

```diff
@@ -70,3 +71,15 @@ case class EntityDataRetriever(config: DataServiceConfig, filters: Seq[String] =
 
 
 }
+
+object EntityDataRetriever {
+  def buildUrl(uri: String, filters: Seq[String]): String = {
+    val parameters = (filters :+ "limit=100").mkString("&")
+    if (uri.contains("?")) {
+      s"$uri&$parameters"
```

Comment:
Should not just be `$uri$parameters`?

Reply:
No the first `&` need to be added. Look at the test